frontend/composables/useMenuEntriesState.ts

```diff
@@ -0,0 +1,77 @@
+import { ref, onMounted, onUnmounted } from "vue";
+import { useRouter, useRoute } from "vue-router";
+
+
+const createMenuEntry = (label: string, basePath: string, iconURL: string, active: boolean) => {
+  const { locale, locales } = useI18n();
+  const router = useRouter();
+  const id = Number(router.currentRoute.value.params.id);
+  const routeURL = `/${locale.value}/${basePath}/${id}/${label.split(".").pop()!.toLowerCase()}`;
+  const selected = useRoute().path.endsWith(label.toLowerCase());
+
+  return {
+    label,
+    routeURL,
+    iconURL,
+    selected,
+    active,
+    id,
+  };
+};
+
+const useMenuEntriesState = () => {
+  const router = useRouter();
+  const currentPath = ref(router.currentRoute.value.fullPath);
+  let removeGuard = () => {};
+
+  const organizationEntries = ref([
+    createMenuEntry("_global.about", "organizations", "bi:card-text", true),
+    createMenuEntry("components.sidebar-left-selector.label.events", "organizations", "bi:calendar-check", true),
```

Comment:
I'm expecting that these i18n keys should also be replaced with `_global` variants :)